app/swagger/swagger/requests/appeals/appeals.rb

```diff
@@ -18,125 +18,66 @@ class Appeals
-              end
+              schema '$ref': :Errors
             end
 
             response 502 do
               key :description, 'Bad Gateway: the upstream appeals app returned an invalid response (500+)'
-              schema do
-                key :'$ref', :Errors
-              end
+              schema '$ref': :Errors
             end
           end
         end
 
-        swagger_path '/v0/appeals/higher_level_reviews' do
+        swagger_path '/v0/higher_level_reviews' do
           operation :post do
             key :tags, %w[higher_level_reviews]
             key :summary, 'Creates a higher level review'
             key :operationId, 'createHigherLevelReview'
-            key :description, 'This endpoint will submit a Decision Review request of type Higher-Level Review. '\
-                              'This endpoint is analogous to submitting VA For 20-0996 via mail or fax. ### '\
-                              'Asynchronous processing The Decision Reviews API leverages a pattern recommended '\
-                              'by JSON API for asynchronous processing (more information '\
-                              '[here](https://jsonapi.org/recommendations/#asynchronous-processing)). '\
-                              'Submitting a Decision Review is an asynchronous process due to the multiple '\
-                              'systems that are involved. Because of this, when a new Decision Review is '\
-                              'submitted, the POST endpoint will return a 202 `accepted` response including an '\
-                              'Intake UUID. That Intake UUID is associated with the submission and can be used '\
-                              'as an identifier for the Intake Status endpoint. While the submission is '\
-                              'processing the Intake Status endpoint will return a 200 response. Once the '\
-                              'asynchronous process is complete and the Decision Review is submitted into '\
-                              'Caseflow, the Intake Status endpoint will return a 303 response including the '\
-                              'Decision Review ID for the submitted Decision Review, which will allow you to '\
-                              'retrieve the full details including the processing status.'
+            description = 'Creates a filled-out HLR PDF and uploads it to Central Mail.' \
+                          ' NOTE: If `informalConference` is false, the fields `informalConferenceRep`' \
+                          ' and `informalConferenceTimes` cannot be present.'
+            key :description, description
 
             parameter do
               key :name, :request
               key :in, :body
-              key :description, 'Higher level review raw request data'
               key :required, true
-
-              schema do
-                key :'$ref', :HigherLevelReviewRequest
-              end
-            end
-
-            response 202 do
```

Comment:
does this mean that the only responses are 200 or 422?

Reply:
for the create? pretty much